pkg/proxy/iptables/proxier.go

```diff
@@ -1168,15 +1125,15 @@ func (proxier *Proxier) syncProxyRules() {
 				// for you.  Since that might bounce off-node, we masquerade here.
 				// If/when we support "Local" policy for VIPs, we should update this.
```

Comment:
oh, I need to update that comment. (It had gone away with the "simplify masquerading" commit before but it's back now...)

Anyway, this PR does not change any behavior here. The behavior of external-to-ClusterIP with Cluster traffic policy has never changed, and since the introduction of `internalTrafficPolicy`, the behavior of external-to-ClusterIP with iTP:Local has (AFAIK) always been that the connection gets Local traffic policy. If you don't want that, then don't use iTP:Local with your externally-accessible ClusterIPs...

Reply:
I guess I am arguing that IPT=Local broke an older semantic and I am not sure if that's OK